apps/librelingo_utils/librelingo_utils/utils.py

```diff
@@ -44,10 +44,10 @@ def clean_word(word: Word):
     """
     Remove punctuation and other special characters from a word.
     """
-    MATCH_NON_WORD_CHARACTERS_BEGINNING = regex.compile("^[^\\w']+")
-    MATCH_NON_WORD_CHARACTERS_END = regex.compile("[^\\w']+$")
-    return MATCH_NON_WORD_CHARACTERS_BEGINNING.sub(
-        "", MATCH_NON_WORD_CHARACTERS_END.sub("", word)
+    match_non_word_characters_beginning = regex.compile("^[^\\w']+")
```

Comment:
could these, instead of being renamed, simply be extracted outside the function? My understanding is that all caps snake case is a convention for constants. I guess, technically these are constants but I imagine they are not recognized as such because they are in a function scope, not in the module scope. So I think if you just move this out of the function as a "global" const, it should work without renaming: `MATCH_NON_WORD_CHARACTERS_BEGINNING = regex.compile("^[^\\w']+")`